smoketest/tests/upgrade_gateway.rs

```diff
@@ -143,20 +141,20 @@ async fn upgrade_gateway() {
 		println!("Polling bridgehub block {} for upgrade event.", block.number());
 		let upgrades = block.events().await.expect("read block events");
 		for upgrade in upgrades.find::<ethereum_control::events::Upgrade>() {
-			let upgrade = upgrade.expect("expect upgrade");
-			assert_eq!(upgrade.impl_address, GATETWAY_UPGRADE_MOCK_CONTRACT.into());
-			assert_eq!(upgrade.impl_code_hash, gateway_upgrade_mock_code_hash.into());
-			assert_eq!(upgrade.params_hash, Some(params_hash.into()));
+			let _upgrade = upgrade.expect("expect upgrade");
+			// assert_eq!(upgrade.impl_address, GATETWAY_UPGRADE_MOCK_CONTRACT.into());
+			// assert_eq!(upgrade.impl_code_hash, gateway_upgrade_mock_code_hash.into());
+			// assert_eq!(upgrade.params_hash, Some(params_hash.into()));
```

Comment:
Can we remove these?

```suggestion
			let _upgrade = upgrade.expect("expect upgrade");
			// assert_eq!(upgrade.impl_address, GATETWAY_UPGRADE_MOCK_CONTRACT.into());
			// assert_eq!(upgrade.impl_code_hash, gateway_upgrade_mock_code_hash.into());
			// assert_eq!(upgrade.params_hash, Some(params_hash.into()));
```

If so, also remove [line 79](https://github.com/Snowfork/snowbridge/pull/940/files#diff-d4bb813697a0a98a0366b54c5ebd604d9572f1b2ff40240eccdadc4455a1918aR79).

Reply:
I see you are still busy with the smoke tests, apologies. 😄